lib/lotus/repository.rb

```diff
@@ -296,10 +249,14 @@ def persist(entity)
       #   created_article = ArticleRepository.create(existing_article) # => no-op
       #   created_article # => nil
       #
+      #   imported_article = ArticleRepository.create(Article.new(title: 'New Lotus features', id: 123))
+      #   imported_article.id  # => 123
       def create(entity)
-        unless _persisted?(entity)
-          _touch(entity)
+        _touch(entity)
+        begin
           @adapter.create(collection, entity)
+        rescue Exception => e
+          raise Lotus::Model::AlreadyPersistedEntityError.new e.message
```

Comment:
is this the right error type in all cases where an _Exception_ can occur from the underlying ```@adapter```?